index.js

```diff
@@ -239,18 +239,8 @@ class LedgerBridgeKeyring extends EventEmitter {
         return tx
       })
     }
-    // For transactions created by newer versions of @ethereumjs/tx
-    // Note: https://github.com/ethereumjs/ethereumjs-monorepo/issues/1188
```

Comment:
This issue was resolved in `@ethereumjs/tx@v3.1.4`, but this package is still using `^3.1.1`. Perhaps it's no longer needed in your testing because you were using the updated package? If so, we should update the range used by this package to ensure v3.1.4 is the minimum.